eZ/Publish/Core/SignalSlot/Tests/ContentServiceTest.php

```diff
@@ -295,6 +296,14 @@ public function serviceProvider()
                 0,
             ),
             array(
+                'removeTranslation',
+                array($contentInfo, $language),
+                null,
+                1,
+                RemoveTranslationSignal::class,
+                array('contentId' => $contentId, 'languageCode' => $language),
```

Comment:
I left this on purpose, because it would imply changing [this entire block](https://github.com/alongosz/ezpublish-kernel/blob/cb209c8346fc8e4567d707313fb5a2e6575777aa/eZ/Publish/Core/SignalSlot/Tests/ContentServiceTest.php#L89-L336) and as discussed it "obfuscates" the diff :)

So I see two solutions:
1. I still can do it only for this one-line inner array. It actually doesn't look _that_ bad.
2. I change whole block in a separate commit and we try to live with this :)

The second solution I applied somewhere before, so I know, I'm already inconsistent ;)

POV ping @Nattfarinn @andrerom